### PR TITLE
rimage: tgl update

### PR DESCRIPTION
**Pull request by jajanusz**
Just for MEU ATM, there is no plain-rimage signing for non-MEU targets in this PR

### CODE REVIEW

File: rimage/man_tgl.c [hunk 1]
```diff
@@ -0,0 +1,126 @@
...
+		.ext_len	= sizeof(struct signed_pkg_info_ext),
+		.name		= "ADSP",
+		.vcn		= 0,
+		.bitmap		= {0, 0, 0, 0, 8},
+
+		.module[0]	= {
+			.name		= "cavs0015.met",
+			.meta_size	= 96,
+			.type		= 0x03,
+			.hash_algo	= 0x02, /* SHA 256 */
```
Comment (lgirdwood) [inline rimage/man_tgl.c]: SHA384 in comment ?

Comment (jajanusz) [reply rimage/man_tgl.c]: This one is still sha256, sha384 is in adsp ext description

**Comment by jajanusz** [comment]
> Looks good, just one question on the comment, needs more detail in commit messages too.
> btw, when will rimage signing part be ready ?

When I'll have platform where I can test it, for now I have only one with BIOS for MEU, so idk.

**Comment by lgirdwood** [comment]
@jajanusz I guess the QB CI TGL failure is due to this update and expected (where CI wil then be updated with new rimage) ?

**Comment by jajanusz** [comment]
> @jajanusz I guess the QB CI TGL failure is due to this update and expected (where CI wil then be updated with new rimage) ?

Yes CI is failing cos TGL in CI has old CSME, we will change it after this PR will be merged.

**Pull request merged**